tests/test_v1.py

```diff
@@ -23,23 +23,38 @@
 import mongomock
 from mongomock.gridfs import enable_gridfs_integration
 import pytest
-import testflinger
-from testflinger.api import v1
+import src
+from src.api import v1
+
+
+class MongoClientMock(mongomock.MongoClient):
+    """Mock MongoClient and allow GridFS"""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        enable_gridfs_integration()
+
+    def start_session(self, *args, **kwargs):
```

Comment:
Isn't it better to disable pylint for specific lines instead of using dirty hacks?

Reply:
I don't really have a strong opinion on whether it's dirtier to ignore the pylint warning or do this, however I think it's quite common that #noqa lines last forever and sometimes without a clear reason. Pylint was complaining that start_session() was an abstract method and that I wasn't implementing it. Looking at MongoClient, I don't *think* that seems to be true from what I'm seeing. However, I also see no harm in implementing the stub to make it happy. I don't call it directly, but perhaps something else might.

Reply:
No problem, if you have a reason to keep it, let's try it!